src/scout_apm/core/traceback.py

```diff
@@ -0,0 +1,21 @@
+# Module for helper functions to capture tracebacks
+
+import traceback
+
+
+def capture():
+    stack = traceback.extract_stack()
+    formatted_stack = []
```

Comment:
Reviewing our [Ruby Backtrace Parser](https://github.com/scoutapp/scout_apm_ruby/blob/f2f84dd67d0a086fe52c0b0c49e0011315717502/lib/scout_apm/utils/backtrace_parser.rb) logic, it looks like we fetch 8 in-app frames (with a regex to determine if it is in-app).

Here's how an example looks right now for a Django ORM call in a view:

![image](https://user-images.githubusercontent.com/7880/39319982-fa19cae0-493f-11e8-9f94-fd5e30c7d2ca.png)